backend/errors/errors.go

```diff
@@ -17,3 +17,9 @@ const (
 	DBConnectionError = "Can't connect to the DB"
 	DBQueryError      = "Error while querying the DB"
 )
+
+
+// Common ES errors
```

Comment:
Is this the only possible error?

Reply:
No, there are other kinds of errors that elastic search can raise. But normally we're treating them as a Generic InternalServerError concatenated with the error message.

I can specify them while building the API, what do you think?